README.md

```diff
@@ -93,7 +93,7 @@ The original Discourse code contributors can be found in [**AUTHORS.MD**](docs/A
 
 ## Copyright / License
 
-Copyright 2014 - 2021 Civilized Discourse Construction Kit, Inc.
+Copyright 2014 - 2022 Civilized Discourse Construction Kit, Inc.
```

Comment:
I don't know, seems risky, could be introducing a lot of technical debt here :P

Reply:
Come on @martin-brennan  we're already in 2022 :P 

https://github.com/discourse/discourse/blob/30a5c01f11f43b1105dc90670e6853ee7272c312/app/assets/javascripts/discourse/app/mixins/upload-debugging.js#L34

Reply:
lol you two